controllers/front/payment.php

```diff
@@ -227,11 +229,12 @@ public function postProcess()
                 ->setType(\Pagantis\OrdersApiClient\Model\Order\Configuration\Channel::ONLINE)
             ;
 
+            $language = $this->getUserLanguage($shippingAddress, $billingAddress);
```

Comment:
change to $purchaseCountry, it could be confused with the other language

Reply:
changed